### PR TITLE
feat: update java workshop content and code to cdk v2

### PR DESCRIPTION
**Pull request by corymhall**
this updates the java workshop content and code to cdk v2.

re #308

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by corymhall** [comment]
Wanted to make sure the builds passed. Final update will bump the `aws-cdk-lib` version to `2.0.0`

File: code/java/main-workshop/src/main/java/com/myorg/CdkWorkshopStack.java [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package com.myorg;
 
-import com.github.eladb.dynamotableviewer.TableViewer;
+import io.github.cdklabs.dynamotableviewer.TableViewer;
```
Comment (njlynch) [inline code/java/main-workshop/src/main/java/com/myorg/CdkWorkshopStack.java:3]: :+1:

**Review by njlynch — COMMENTED** [review]
Nice!

**Comment by corymhall** [comment]
Fixed in #358

**Pull request closed without merging**